Library/Homebrew/dev-cmd/test.rb

```diff
@@ -45,6 +45,15 @@ def test
         next
       end
 
+      # Don't test formulae missing test dependencies
+      missing_test_deps = f.recursive_dependencies do |_, dependency|
+        Dependency.prune if !dependency.required? && !dependency.test?
+      end.map(&:to_s)
+      unless missing_test_deps.empty?
+        ofail "#{f.full_name} is missing test dependencies: #{missing_test_deps.join(" ")}"
+        next
```

Comment:
Should probably say what can be done about this (run `brew reinstall #{f.full_name} --include-test`)? Or even install the test dependencies tbh.

Reply:
@alyssais I think it should be obvious enough for anyone running `brew test` but thanks 👍